### ISSUE TITLE
Extra space above quick form tab content

### ISSUE THREAD

**Issue by paul121** [Rothamsted-Ecoinformatics/farm_rothamsted#208]
We suspect that a recent `drupal/gin` or a `drupal/field_group` update changed some styling that is adding additional space above the content in quick form tabs.

**Comment by paul121** [comment]
This was fixed in the first commit after the release of 3.0-beta3: https://www.drupal.org/project/gin/issues/3282294

Pinning drupal/gin to this commit fixes the issue. I'll open a new issue to depend on at least ^3.0-beta4 once they release a new version.

@aislinnpearson they also fixed a related issue for vertical tabs which will add a red `*` to the vertical tabs which have required fields. See examples in the attached .png files: https://www.drupal.org/project/gin/issues/3279752 We will get this feature with 3.0-beta4 :-)

**Comment by mstenta** [comment]
Sorry @paul121 but I'd like to see if @aislinnpearson would be open to waiting for Gin to release a new version for this to be fixed.

I ran into the issue you mentioned on our call when I tried to build the new image for deployment:

```
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Root composer.json requires farmier/farm_rothamsted 2.3.0 -> satisfiable by farmier/farm_rothamsted[2.3.0].
    - farmier/farm_rothamsted 2.3.0 requires drupal/gin 3.x-dev#7a115a3750421c0e94fb5e779cc554b815050c16 -> found drupal/gin[dev-3.x, 3.x-dev (alias of dev-3.x)] but the package is fixed to 3.0.0-beta3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.
```

This was with `composer install`. I don't use `composer update` at all in my build pipeline, and I'm not familiar enough with the `--with-all-dependencies` option to feel comfortable using it.

Since this is a relatively minor issue, fixed upstream, with a good chance of a new Gin release in the near future, I don't know if it justifies the effort to figure this out on my end right now.

Is that OK @aislinnpearson? Can you live with the extra spacing for a bit until the Gin project makes a new release? Or if not, @paul121 would a CSS hot-fix in this module be possible?

Sorry to reopen and add complexity. Just don't have the time to figure out how to detangle the build process on my end this week. :-/

**Comment by mstenta** [comment]
I figured out another way to do this. See https://github.com/Rothamsted-Ecoinformatics/farm_rothamsted/issues/218#issuecomment-1156896452
